docs/sections/external_links.rst

```diff
@@ -0,0 +1,8 @@
+.. _anaconda: https://anaconda.org/maddenp/condev  
```

Comment:
Simple addition to break up the reusability:

```
.. _External Links:
.. _anaconda: https://anaconda.org/maddenp/condev .
.. _black: https://github.com/black-software-Com/Black-Tool
.
.


.. _Formatting Replacement:
.. |black| replace:: ``black``
.. |isort| replace:: ``isort``
.
.
```

Reply:
We probably don't want to link "anaconda" to the condev repo -- https://anaconda.org/ would probably be the right link for that. A good link for condev in general would be its GitHub page https://github.com/maddenp/condev (which you've got, below), and for pre-build condev packages, https://anaconda.org/maddenp/condev

Reply:
Following up on some clarification on this comment, I believe I have the correct link you mentioned:
https://anaconda.org/maddenp/condev

Is this a change you want in the documentation, where currently the text mentions "anaconda". The text link to and the name that holds the link need to match appropriately or the hyperlink will not be found correctly. I'll highlight in another comment which line i'm talking about in the documentation.

Reply:
Oh, I think maybe I understand: Is this different than the kind of `[some text](https://the.link)` that we'd often use in docs for hyperlinks, where we can have arbitrary text and then the link? I mean, in other docs we might have the same link but used with different text, like `[Google](https://google.com) is a search engine` and then `You can search for that [here](https://google.com)`, where we've chosen to put the link on the text `Google` in the first case and on `here` in the second case. I think we might want to be able, in our RST docs, to have something besides a single, special word linked to a web page. I was thinking (still using Markdown syntax here) of pointing users to the most appropriate page for the context, for example
```
As an alternative to installing [a prebuilt package from anaconda](https://anaconda.org/maddenp/condev), the...
```

for the case where it'd be useful for the user to land on the specific anaconda.org page for condev packages,
```
Install the [condev](https://github.com/maddenp/condev) package into the base environment.
```

(which actually already looks good in the PR) for the case where knowing more about condev might be useful, such that its GitHub page might be a good resource. I don't currently see a place where it would actually make sense to link to https://anaconda.org, now that I'm really looking.

But in the first case, it's not just a single keyword that we're using for the link, but a phrase. I think it'd be unwieldy to try to factor every phrase out to an external links file, but I do think we'll need to link from arbitrary phrases so I think I'd vote in favor of a little URL duplication if necessary, if the URL needs to live in the main text so we can turn a phrase into a hyperlink.

I hope that makes sense, but let's discuss in group if not!

Reply:
> Oh, I think maybe I understand: Is this different than the kind of `[some text](https://the.link)` that we'd often use in docs for hyperlinks, where we can have arbitrary text and then the link? I mean, in other docs we might have the same link but used with different text, like `[Google](https://google.com) is a search engine` and then `You can search for that [here](https://google.com)`, where we've chosen to put the link on the text `Google` in the first case and on `here` in the second case. I think we might want to be able, in our RST docs, to have something besides a single, special word linked to a web page. I was thinking (still using Markdown syntax here) of pointing users to the most appropriate page for the context

I'm going to have to think about this case a bit more as I haven't thought about it before. This seems like a great use case and some more research on it would be beneficial. 

There is currently a way to do arbirturary text and then a link similar to what is in markdown:
``a prebuilt package from anaconda <https://anaconda.org/madde    np/condev>`_, ...`

I added that in for now in replace of the previous anaconda text.